test/scalacheck/scala/collection/StringOpsProps.scala

```diff
@@ -0,0 +1,32 @@
+
+package scala.collection
+
+import java.io.{BufferedReader, StringReader}
+
+import org.scalacheck.{Gen, Properties}, Gen.{oneOf, listOf}
+import org.scalacheck.Prop._
+
+import JavaConverters._
+
+object StringOpsTest extends Properties("StringOps") {
+
+  val lineChar: Gen[Char] = oneOf('X', '\r', '\n')
```

Comment:
form feed?

Reply:
`\f` will never be treated the same by java API, not that I've tried it.

I don't know why Odersky says that Scala is characterized by its use of implicits. Scala is the language that splits on form feed!

Reply:
oh, I misunderstood, thought you were including FF because Java does.

why is it even desirable to include FF? why not just match Java?

Reply:
If we're not required to do it, we should skip it.  The fewer checks, the faster.

Reply:
There is an emacs convention about grouping code. I think that is the origin, but I didn't research it. I use vi.

Reply:
Also, my daughter was in the spelling bee today and messed up on Prague for some reason, so I agree, the fewer Czechs, the better.